Formula/zet.rb

```diff
@@ -0,0 +1,22 @@
+class Zet < Formula
+  desc "CLI utility to find the union, intersection, and set difference of files"
+  homepage "https://github.com/yarrow/zet"
+  url "https://github.com/yarrow/zet/archive/refs/tags/0.2.0.tar.gz"
+  sha256 "b001632ecff545411908a9b365dbac6f930e563233547a4cb0ad210d3066952b"
+  license any_of: ["Apache-2.0", "MIT"]
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    (testpath/"foo.txt").write("1\n2\n3\n4\n5\n")
+    (testpath/"bar.txt").write("1\n2\n4\n")
+    resource("testdata").stage do
+      assert_equal "3/n5/n",
+      shell_output("#{bin}/zet diff foo.txt bar.txt")
+    end
```

Comment:
```suggestion
    assert_equal "3/n5/n", shell_output("#{bin}/zet diff foo.txt bar.txt")
```
I'm not even sure why you need a `resource` here. Doesn't this work?